docs/literate/make.jl

```diff
@@ -75,7 +75,16 @@ function create_tutorials(files)
     end
 
     # Generate markdown file for introduction page
-    Literate.markdown(joinpath(repo_src, "index.jl"), pages_dir; name="introduction")
+    # Preprocessing introduction file: Generate tutorial numbers
```

Comment:
```suggestion
    # Preprocessing introduction file: Generate consecutive tutorial numbers by replacing
    # each occurrence of `$index` with an integer incremented by 1, starting at 1.
```

Reply:
Done in [b88f128](https://github.com/trixi-framework/Trixi.jl/pull/1901/commits/b88f12841b3156045ff535053c255fd1eaf0d02e).